packages/teleport/src/cluster/components/Nodes/Nodes.tsx

```diff
@@ -51,6 +55,15 @@ export function Nodes({
     <FeatureBox>
       <FeatureHeader alignItems="center">
         <FeatureHeaderTitle mr="5">Nodes</FeatureHeaderTitle>
+        <ButtonPrimary
+          ml="auto"
+          width="240px"
+          as="a"
+          href={cfg.getConsoleNodesRoute(clusterId)}
+          target="_blank"
+        >
+          New Session
+        </ButtonPrimary>
```

Comment:
![image](https://user-images.githubusercontent.com/43280172/84548041-a5465a80-acb9-11ea-9da7-9706e4be7874.png)

Reply:
Maybe we should make a component for this button, it is used in 3 places now with same styling:
```
margin-left: auto
width: 240px
```

Reply:
With that said, every where else, we use the purple icon as above, we should probably convert this button below to same styling to keep it consistent?

![image](https://user-images.githubusercontent.com/43280172/84548528-a88e1600-acba-11ea-8714-91b8eb228d04.png)

Reply:
this is TBD.